### PR TITLE
miniz: silence warnings

### PR DESCRIPTION
**Pull request by tautschnig**
This is third-party code.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] My contribution is formatted in line with CODING_STANDARD.md.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 00fedc5964d066a851dc8b77d946e291b3406690).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85961450

File: jbmc/src/miniz/miniz.h [hunk 1]
```diff
@@ -173,6 +173,10 @@
 #pragma once
 
 
+#ifdef _MSC_VER
```
Comment (smowton) [inline jbmc/src/miniz/miniz.h:176]: This leaks into the scope of anyone including miniz.h -- if your intent is scope of this header file, use pragma-push / pragma-pop to scope it. (the .cpp file one is fine, implicitly scoped to the translation unit)

Comment (tautschnig) [reply jbmc/src/miniz/miniz.h:176]: Done.

**Review by smowton — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 1f48810378612bcfc93e5fe5f593d084b951e5c3).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85988754

**Pull request merged**